### DISCUSSION TITLE
Configure HTTP probe to not follow 301 redirect?

### DISCUSSION THREAD

**Discussion by ypakhomenko** [cloudprober/cloudprober#413] [Q&A] [answered]
Hi everyone,

Sorry, did not find it in the docs, is it possible to configure HTTP probe to not follow redirects? 
I just want to check if I get 301 within some time frame. 
And following redirect results in timeout because the new location FQDN cannot be resolved by the proxy DNS and proxy returns 503 ERR_DNS_FAIL after ~5 seconds.

Thanks && Best regards,
Yury

**Comment by manugarg** [comment, marked as answer]
I think `max_redirects` option is what you need:
https://github.com/cloudprober/cloudprober/blob/312081c3e5bb0dcfe74e266724646a40af2809c9/probes/http/proto/config.proto#L117

**Reply by ypakhomenko** [reply]
Exactly, thanks a lot!
